src/components/language-selector.jsx

```diff
@@ -12,18 +12,27 @@ function LanguageSelector({ languageList, slug }) {
 
   const [isOpen, setIsOpen] = useState(false);
 
+  /* when a language button is clicked, it navigates to the url
+  with the language code of the clicked button and the slug of the current page */
   const handleClick = (langKey) => {
     i18n.changeLanguage(langKey, navigate(`/${langKey}/${slug}`));
   };
 
+  // create the language selector buttons based on the language list (languages used in src/locales)
+  // for 5 or less languages, the buttons are simply in the banner at the top of the page
+  // for 5 or more languages, the buttons are in a dropdown menu
   const languageMenuItems = languageList.map((language) => (
     <div
       className={
-        languageList.length <= 5
-          ? 'banner__language-selector-item'
-          : i18n.dir(language.langKey) === 'rtl'
-            ? 'banner__language-selector-item--rtl'
-            : 'banner__language-selector-item--ltr'
+        `${languageList.length <= 5 // are there are less than 5 languages used in the locales folder
+          ? 'banner__language-selector-item' // less than 5 languages
+          : i18n.dir(language.langKey) === 'rtl' // if more than 5 languages, check if the language is right-to-left
+            ? 'banner__language-selector-item--rtl' // if right-to-left the language buttons will have slightly different styles
```

Comment:
// if right-to-left, the language buttons will have slightly different styles

I know what you're saying here, but it may not be as obvious for others.

Reply:
Changed the phrasing a bit, in the styles files there is also more information on how the styles differ for rtl or ltr